templates/site/node.html

```diff
@@ -8,7 +8,7 @@
                 <div class="row">
                     <div class="col-9">
                         <!--H2 title -->
-                        <div id="name" class="mb-0"></div>
+                        <div id="display_name" class="mb-0"></div>
```

Comment:
I don't think this div is ever filled with anything.... is changing the id necessary? Should we get rid of this div or is there supposed to be something here?

Reply:
I don't remember what it was originally for.  Best I can guess is it's supposed to fill in the node's name for the title (Red Oak@University of Toronto).  

If it's not used I'll take it out.